src/test/java/helpers/UserCreator.java

```diff
@@ -0,0 +1,20 @@
+package helpers;
+
+import model.User;
+
+public class UserCreator {
+
+    public static final String TESTDATA_USER_NAME = ConfigurationManager.getProperty("test.data.user.name");
+    public static final String TESTDATA_USER_PASSWORD = ConfigurationManager.getProperty("test.data.user.password");
+    public static final String LOCKEDOUT_USER_NAME = ConfigurationManager.getProperty("lockedout.user.name");
+
+    public static User withStandardCredentials() {
```

Comment:
actually using the verb in the method name is preferable. I recommend to add UserType enum here and rename method e.g. getUser (UserType userType). It is close to Factory. UserManager class name be more suitable in this case